arch/arm/boot/dts/bcm2708-rpi-bt.dtsi

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0
+
+&uart0 {
+	bt: bluetooth {
+		compatible = "brcm,bcm43438-bt";
+		max-speed = <3000000>;
+		shutdown-gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
```

Comment:
Shouldn't this be GPIO 45? (I'm on a phone screen so might have missed something)

Reply:
Yes, you are absolutely right. Fixing.